CMakeLists.txt

```diff
@@ -32,6 +32,15 @@ set(MLIR_MAIN_SRC_DIR ${LLVM_MAIN_SRC_DIR}/../mlir)
 set(MLIR_INCLUDE_DIR ${LLVM_MAIN_SRC_DIR}/../mlir/include)
 set(MLIR_TABLEGEN_OUTPUT_DIR ${CMAKE_BINARY_DIR}/tools/mlir/include)
 
+# Disable experimental alp by default
+set(SANDBOX_ENABLE_ALP OFF)
```

Comment:
I think this will always override whatever you give when calling the cmake command?

Reply:
Good point!